all-repos.yaml

```diff
@@ -220,3 +220,4 @@
 - https://github.com/cmhughes/latexindent.pl
 - https://github.com/sirwart/ripsecrets
 - https://github.com/bagerard/graphviz-dot-hooks
+- https://github.com/kwongtn/pipfile-sort
```

Comment:
this looks like an unofficial fork?

Reply:
@asottile It is, I did try to make the original package pre-commit friendly by submitting a PR, but the author did not respond to anything 😅
https://github.com/msh5/pipfile-sort/pull/23

Reply:
I would maybe wait a little bit more, sometimes PRs can take months to get noticed